api/server/routes_create.go

```diff
@@ -44,13 +43,13 @@ func (s *Server) handleRouteCreate(c *gin.Context) {
 		return
 	}
 
-	err = s.Runner.EnsureImageExists(ctx, &task.Config{
-		Image: wroute.Route.Image,
-	})
-	if err != nil {
-		c.JSON(http.StatusBadRequest, simpleError(models.ErrUsableImage))
-		return
-	}
+	// err = s.Runner.EnsureImageExists(ctx, &task.Config{
+	// 	Image: wroute.Route.Image,
+	// })
+	// if err != nil {
+	// 	c.JSON(http.StatusBadRequest, simpleError(models.ErrUsableImage))
+	// 	return
+	// }
```

Comment:
Going to make a new issue to follow up on this. Currently if the image isn't found, the response to the user is a blank 200. Created #540 to follow up on this.